### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by szepeviktor**
## Description

Found few misspellings.

### CODE REVIEW

File: includes/class-wc-subscriptions-manager.php [hunk 1]
```diff
@@ -1852,9 +1852,9 @@ public static function array_merge_recursive_for_real( $first_array, $second_arr
 	 */
 	public static function get_amount_from_proportion( $total, $proportion ) {
 
-		$sign_up_fee_proprotion = 1 - $proportion;
+		$sign_up_fee_proportion = 1 - $proportion;
 
-		$sign_up_total    = round( $total * $sign_up_fee_proprotion, 2 );
+		$sign_up_total    = round( $total * $sign_up_fee_proportion, 2 );
```
Comment (james-allan) [inline includes/class-wc-subscriptions-manager.php:1857]: 👍 Noting that I double checked those are only 2 uses of this variable.

File: includes/class-wc-subscriptions-order.php [hunk 2]
```diff
@@ -2170,9 +2170,9 @@ public static function get_outstanding_balance( $order, $product_id ) {
 
 		$failed_payment_count = self::get_failed_payment_count( $order, $product_id );
 
-		$oustanding_balance = $failed_payment_count * self::get_recurring_total( $order, $product_id );
+		$outstanding_balance = $failed_payment_count * self::get_recurring_total( $order, $product_id );
 
-		return $oustanding_balance;
+		return $outstanding_balance;
```
Comment (james-allan) [inline includes/class-wc-subscriptions-order.php:2175]: 👍 Noting that I double checked these are the only 2 uses of this variable.

This is also a deprecated function.

File: includes/class-wcs-template-loader.php [hunk 3]
```diff
@@ -364,7 +364,7 @@ public static function get_opc_subscription_add_to_cart() {
 	 * @since 1.4.0
 	 *
 	 * @param string $template
-	 * @param string $tempalte_name
+	 * @param string $template_name
```
Comment (james-allan) [inline includes/class-wcs-template-loader.php:367]: 👍 I checked that the variable in the actual function signature is correct.

File: includes/upgrades/class-wcs-upgrade-1-2.php [hunk 4]
```diff
@@ -197,8 +197,8 @@ public static function init() {
 
 					// Base price * Quantity
 					$sign_up_fee_line_subtotal     = WC_Subscriptions_Order::get_meta( $order, '_cart_contents_sign_up_fee_total', 0 ) + WC_Subscriptions_Order::get_meta( $order, '_sign_up_fee_discount_cart', 0 );
-					$sign_up_fee_propotion         = ( $sign_up_fee_line_total > 0 ) ? $sign_up_fee_line_subtotal / $sign_up_fee_line_total : 0;
-					$sign_up_fee_line_subtotal_tax = WC_Subscriptions_Manager::get_amount_from_proportion( WC_Subscriptions_Order::get_meta( $order, '_sign_up_fee_tax_total', 0 ), $sign_up_fee_propotion );
+					$sign_up_fee_proportion        = ( $sign_up_fee_line_total > 0 ) ? $sign_up_fee_line_subtotal / $sign_up_fee_line_total : 0;
+					$sign_up_fee_line_subtotal_tax = WC_Subscriptions_Manager::get_amount_from_proportion( WC_Subscriptions_Order::get_meta( $order, '_sign_up_fee_tax_total', 0 ), $sign_up_fee_proportion );
```
Comment (james-allan) [inline includes/upgrades/class-wcs-upgrade-1-2.php:201]: I've also double checked these are the only 2 uses of this variable too. 👍

File: tests/unit/test-class-wc-subscriptions.php [hunk 5]
```diff
@@ -112,7 +112,7 @@ public function test_can_be_updated_to_active() {
 		$this->assertEquals( $expected, $can_be_updated, '[FAILED]: pending-cancel to wc-active.' );
 
 		// End date in the past
-		$subcription->update_dates( [ 'end' => gmdate( 'Y-m-d H:i:s', time() - DAY_IN_SECONDS ) ] );
```
Comment (james-allan) [inline tests/unit/test-class-wc-subscriptions.php:115]: This and the 2 updates above are the only 3 uses of it. It's also in unit tests so not a concern. 👍

**Review by james-allan — APPROVED** [review]
Thanks for the PR @szepeviktor. 💯

File: tests/unit/test-class-wc-subscriptions.php (hunk 5 above)
Comment (szepeviktor) [reply tests/unit/test-class-wc-subscriptions.php:115]: Thank you. I've also checked all variable uses.

**Comment by szepeviktor** [comment]
Glad to contribute.

**Pull request merged**